gromacs/gromacs-2022.x/colvarproxy_gromacs.cpp

```diff
@@ -95,7 +95,7 @@ void colvarproxy_gromacs::init(t_inputrec *ir, int64_t step, const gmx_mtop_t &m
   updated_masses_ = updated_charges_ = true;
 
   // GROMACS timestep
-  timestep = ir->delta_t;
+  set_integration_timestep(ir->delta_t * 1000.0);
```

Comment:
I would keep the comment saying GROMACS is uses picoseconds  for the time step hence the 1000 factor.